tests/Doctrine/Tests/DBAL/Types/DateTest.php

```diff
@@ -3,64 +3,31 @@
 namespace Doctrine\Tests\DBAL\Types;
 
 use Doctrine\DBAL\Types\Type;
-use Doctrine\Tests\DBAL\Mocks\MockPlatform;
 
-class DateTest extends \Doctrine\Tests\DbalTestCase
+class DateTestCase extends BaseDateTypeTestCase
```

Comment:
Doesn't matter if nobody extends this (probably nobody will) but the class name doesn't match the filename anymore.